content/hi/docs/tutorials/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: ट्यूटोरियल
+main_menu: true
+no_list: true
+weight: 60
+content_type: concept
+---
+
+<!-- overview -->
+
+प्रलेखन के इस खंड में ट्यूटोरियल हैं।
+ट्यूटोरियल दिखाता है कि किसी एकल कार्य से बड़े [लक्ष्य](/docs/tasks/) को कैसे पूरा किया जाए। 
```

Comment:
```suggestion
ट्यूटोरियल दिखाता है कि कैसे एक कार्य को बड़े [लक्ष्य](/docs/tasks/) में पूरा किया जाए।
```

I am not sure about the last `में पूरा किया जाए।` part

Reply:
@mittalyashu After going through the `en` version, I think `ट्यूटोरियल दिखाता है कि किसी एकल कार्य से बड़े लक्ष्य को कैसे पूरा किया जाए।` is the correct translation.

Reply:
We've two `किसी` and `कैसे`, which sounds quite weird in a sentence.

Reply:
How about `ट्यूटोरियल दिखाता है कि एकल कार्य से बड़े लक्ष्य को कैसे पूरा किया जाए।`

Reply:
i think this is good `ट्यूटोरियल दिखाता है कि किसी एकल कार्य से बड़े [लक्ष्य](/docs/tasks/) को कैसे पूरा किया जाए।`